staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto

```diff
@@ -545,6 +551,113 @@ message ListPodSandboxResponse {
     repeated PodSandbox items = 1;
 }
 
+message PodSandboxStatsRequest {
+    // ID of the pod for which to retrieve stats.
+    string pod_sandbox_id = 1;
+}
+
+message PodSandboxStatsResponse {
+    PodSandboxStats stats = 1;
+}
+
+// PodSandboxStatsFilter is used to filter containers.
```

Comment:
s/is used to filter containers/is used to filter list of pod sandboxes from which to receive stats

Reply:
this shouldn't be containers right?